src/containers/JudgingPanel.js

```diff
@@ -46,16 +47,67 @@ class Project {
   }
 }
 
+const getGradedProjects = async () => {
+  const projectDocs = await projectsRef.get()
+  const projectData = projectDocs.docs.map(projectDoc => {
+    const project = projectDoc.data()
+    if (project.grades) {
+      project.countGraded = Object.values(project.grades).length
+      Object.values(project.grades).forEach(grade => {
+        Object.entries(grade).forEach(([key, value]) => {
+          if (key === 'notes') return
+          project[key] = project[key] ? project[key] + value : value
+          project.total = project.total ? project.total + value : value
+        })
+      })
+      project.grade = project.total / project.countGraded / 5
+      project.tech = project.tech / project.countGraded
+      project.design = project.design / project.countGraded
+      project.functionality = project.functionality / project.countGraded
+      project.creativity = project.creativity / project.countGraded
+      project.pitch = project.pitch / project.countGraded
```

Comment:
can probably be rewritten to use a reduce which would be a lot cleaner (ok for now cause we're time crunched, but maybe add a TODO and make a ticket)

Reply:
how do write a reduce for average?

Reply:
note: am not iterating thru project because project is populated with all kinds of non grading related data such as devpostUrl, description etc

Could be refactored so it doesn't but i'll save that for later